mypy.ini

```diff
@@ -0,0 +1,3 @@
+[mypy]
+ignore_missing_imports = True
+disable_error_code = misc,attr-defined, import
```

Comment:
I understand the value of `ignore_missing_imports`. What kind of errors/warnings are we ignoring with ` disable_error_code`? Do these warnings/errors prevent the MyPy validation from completing successfully?

Also add a newline at the end of the file.

Reply:
Yes MyPy fails without that ignoring

![image](https://user-images.githubusercontent.com/60013703/194543648-933b8c85-7c78-47fa-ad33-e5889b85239e.png)

Reply:
I'd rather not disable these warnings globally. They probably can be adressed locally. If not, let's check if they can be disabled locally too.

For example, I am pretty sure the `[misc]` errors can easily be fixed by reordering the variable declarations in `report.py`. PyCharm (the IDE I am using) highlights these errors and makes it easy to confirm everything is in the right order without having to run `mypy` again.

Reply:
`misc` errors are fixed